### ISSUE TITLE
No video transcode information in history 

### ISSUE THREAD

**Issue by coneslayer** [tfonfara/RemoteForTautulli-Issues#11]
### New Issue Checklist

- [x] My issue is related to the [Remote for Tautulli](https://itunes.apple.com/app/id1343177312) app, report issues for Tautulli [here](https://github.com/Tautulli/Tautulli-Issues/issues)
- [x] I updated Remote for Tautulli to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/tfonfara/RemoteForTautulli-Issues/issues)

### Description

In the History entries, video transcode information is not shown. For example, I have recent history entries where video was transcoded HEVC->H.264, and this is shown in the Tautulli web app, but only the container and audio transcode information is shown in Remote For Tautulli.

### Steps to Reproduce

1. Complete a Plex stream that requires a video transcode 

**Expected behavior:**

History will include video transcode information 

**Actual behavior:**

No video transcode information appears

**Reproduces how often**

Seems 100% for my files/streams

### Environment

* Device: iPhone 7
* OS version: 12.0.1
* App version: 1.7.0
* Tautulli version: 2.1.23-beta

### Additional Information

I can provide screenshots or additional information on request.

**Comment by tfonfara** [comment]
I'm not sure if I understood you correctly. Please see the attached screenshot of a completed stream in history details, video transcode information is shown there:


![](https://user-images.githubusercontent.com/728683/47658006-805bad00-db92-11e8-8bae-9814d847388d.png)

**Comment by coneslayer** [comment]
Yep, that’s what I’m talking about. The video information is missing for me even though it’s recorded by Tautulli and visible in the web interface.

![GitHawk Upload by coneslayer](https://i.imgur.com/oYXN7gz.jpg)

![GitHawk Upload by coneslayer](https://i.imgur.com/GXFEsx4.jpg)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by coneslayer** [comment]
Note: I am getting "Video: Direct Play" entries for recent sessions when that's the case. As far as I can tell, it's only video transcode sessions that are missing the video section.

**Comment by tfonfara** [comment]
I tried a lot of different scenarios, also with exactly same configuration as you (iPhone, iOS version, Tautulli version...), but I'm not able to reproduce this issue, it always shows up video transcoding 😕

**Comment by coneslayer** [comment]
Thanks for taking the time to look into it. My Tautulli server is Internet-reachable, so if you want to access it to debug further, I can support that. But if I'm the only one having this problem, I understand if it's not productive to look into it further.
